packages/admin/src/Commands/MakeUserCommand.php

```diff
@@ -17,14 +17,18 @@ class MakeUserCommand extends Command
 
     protected $description = 'Creates a Filament user.';
 
-    protected $signature = 'make:filament-user';
+    protected $signature = 'make:filament-user {--name= : The name of the user} {--email= : A valid and unique email address} {--password= : The password for the user (min. 8 characters)}';
+
+    protected $options;
+
+
 
     protected function getUserData(): array
     {
         return [
-            'name' => $this->validateInput(fn () => $this->ask('Name'), 'name', ['required']),
-            'email' => $this->validateInput(fn () => $this->ask('Email address'), 'email', ['required', 'email', 'unique:' . $this->getUserModel()]),
-            'password' => Hash::make($this->validateInput(fn () => $this->secret('Password'), 'password', ['required', 'min:8'])),
+            'name' => $this->validateInput(fn () => $this->options['name']??$this->ask('Name'), 'name', ['required'], function () { unset($this->options['name']);}),
+            'email' => $this->validateInput(fn () => $this->options['email']??$this->ask('Email address'), 'email', ['required', 'email', 'unique:' . $this->getUserModel()], function () { unset($this->options['email']);}),
+            'password' => Hash::make($this->validateInput(fn () => $this->options['password']??$this->secret('Password'), 'password', ['required', 'min:8'], function () { unset($this->options['password']);})),
```

Comment:
/wtd Correct spacing between the null operator. Convert the closures to short closures.